command/peers/peers_add.go

```diff
@@ -57,7 +57,7 @@ func (p *PeersAdd) Run(args []string) int {
 	flags := p.FlagSet(p.GetBaseCommand())
 
 	var passedInAddresses = make(helperFlags.ArrayFlags, 0)
-	flags.Var(&passedInAddresses, "addr", "")
+	flags.Var(&passedInAddresses, "Addr", "")
```

Comment:
Did you make this parameter capital by accident?
```suggestion
	flags.Var(&passedInAddresses, "addr", "")
```

Reply:
I do all my find-and-replaces manually, and IntelliJ still somehow manages to sneak in a rogue replace.
Fixed here, and in other places it changed 👍